io/src/main/scala/sbt/internal/io/Milli.scala

```diff
@@ -56,7 +56,7 @@ private abstract class StatInt(size: Int, mtimeOffset: Int, mtimensecOffset: Int
   def getModifiedTimeNative = (buffer.getInt(mtimeOffset), buffer.getInt(mtimensecOffset))
 }
 
-private abstract class Milli {
+private[sbt] abstract class Milli {
```

Comment:
I am of two minds on this one.
I thought we discussed this type of things previously and said we should keep internal code to each modules, and essentially copy-paste so they are less coupled.

Reply:
I see nothing wrong in exposing internal code to `[sbt]`, it is still confined. Code duplication, however, is almost never a good idea.
This PR has my LGTM